src/services/auth/tokens/deleteExpiredTokens.ts

```diff
@@ -0,0 +1,17 @@
+import { prisma } from '@/lib/prisma';
+
+//ggs gonna have to find a way to periodically run this script man (it's a production issue :D )
+async function deleteExpiredTokens() {
+  try {
```

Comment:
Will have to find a way to trigger this via query or periodically run this (TBD)